include/experimental/__p2642_bits/layout_padded.hpp

```diff
@@ -95,6 +95,7 @@ struct padded_extent {
   using static_array_type = typename static_array_type_for_padded_extent<
       padding_value, _Extents, _ExtentToPadIdx, _Extents::rank()>::type;
 
+  MDSPAN_INLINE_FUNCTION
```

Comment:
Do you actually need this or did you just add for consistency?

Reply:
`static_value` is called inside a couple constructors and `is_always_exhaustive` all of which are device callable. The "INLINE" is just for consistency, at some point we can go wholesale through this stuff and replace it all with MDSPAN_FUNCTION